### PR TITLE
Capstone: Spectra-Beaute-Website

### PR DESCRIPTION
**Pull request by Gleennkar**
## Created A Website for Spectra Beaute
- Added the index.html and about.html section
- Used Bootstrap and Scss to display the various products 
- Added some style.css to the index.html and about section
- Used index.js to give the hamburger menu functionality
- Update my README.md explaining the setup
- Run for linter errors
- Link explaining my project https://www.loom.com/share/fd09cfff3af749b580c229b9db31751f

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+      <h1>Welcome to Spectra Beauté</h1>
+      <h2 class="intro-message">"We believe in our responsibility to inspire"</h2>
+    </div>
+    <div class="intro-desc">
+      <p>
+        A source of inspiration and well-being. When you hear of chic, professionalism, perfection, and outstanding
+        customer service, Spectra comes to mind. It is the number one leading beauty spa in Cameroon and Central Africa
+        as a whole. With all the chaos happening in our lives, Spectra is here to give you that peace and a relaxing
+        environment.
+      </p>
```
Comment (rotimiazeez) [inline index.html:93]: - Kindly fix the overlay of this card for a better UI on the mobile screen 😉 
screenshot (3)

File: about.html [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+     <section class="about">
+        <div class="about-content">
+            <h1>Welcome to Spectra Beauté</h1>
+          <h2 class="message">"We believe in our responsibility to inspire"</h2>
+        </div>
+        <div class="Description">
+          <p>
+            Spectra Beauté is considered by most as the leading beauty and wellness salon/spa in Cameroon (and arguably Central Africa). A source of inspiration and wellbeing and my love for this brand is embedded through this creative web-design and the joy, peace i feel when using its exceptional and unique products.
+          </p>
```
Comment (rotimiazeez) [inline about.html:102]: - Kindly fix the overlay on the mobile screen for this description 👍🏽 
screenshot

File: about.html [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+                </ul>
+            </div>          
+        </nav>
+     </div>  
+
+     <section class="about">
+        <div class="about-content">
+            <h1>Welcome to Spectra Beauté</h1>
+          <h2 class="message">"We believe in our responsibility to inspire"</h2>
+        </div>
```
Comment (rotimiazeez) [inline about.html:98]: - Kindly fix the overflow on the page for a better User Interface
screenshot (1)

File: about.html [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+      </section>  
+      
+      <footer id="foot">
+        <div class="footer-logo">
+          <img class="footer-img1" src="./Images/Spectra-logo.webp" alt="Spectra logo">
+        </div>
+        <div class="footer-message-msg">
+          <p>@Spectra Beauté will always be there for you to give you the peace and comfort you deserve</p>
+          <p>A source od inspiration and wellbeing Spectra Beauté</p>
+        </div>
```
Comment (rotimiazeez) [inline about.html:171]: - Please make sure your design matches that of the original design as seen below.
Your design | Behance design
screenshot (2) | screenshot (4)

Comment (rotimiazeez) [reply about.html:171]: - Same with the home page 😄

File: js/index.js [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+  },
+  {
+    id: 4,
+    img: './Images/Gel-Scrub.jpeg',
+    title: 'Gel Scrub',
+    supportingText: 'This soothes and eliminates buildup of dead skin, moisturizes and nourishes while regenerating the skin back to a healthy state',
+    button: 'Buy Now',
+
+  },
```
Comment (rotimiazeez) [inline js/index.js:42]: - Note that for better UI, your design has to match that of the original template
Your design | Behance design
screenshot (6) | screenshot (5)

File: index.html [hunk 6]
```diff
@@ -0,0 +1,277 @@
...
+      <div class="media col-12 col-md-6 mt-4">
+        <img src="./Images/Base-coat.jpeg" alt="Base coat" class="roundimg">
+        <div class="media-body">
+          <h5>Base coat</h5>
+          <p>Base coat helps the gel or the polish stick and prevents it from coming in contact with our natural nails
+          </p>
+          <button type="button" class="btn btn-success">Buy Now</button>
+        </div>
+      </div>-->
+    <!-- </div> -->
```
Comment (rotimiazeez) [inline index.html]: - Kindly remove all commented code for a cleaner and more readable code 👍🏽

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @Gleennkar  👏🏽, 
## Good job so far! 👍🏽 
### Summary ♨️ 
- No Linter error ✔️ 
- Good and descriptive PR 💯 
- Good commit messages
- Good Desktop Ui
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights


## Required Changes ♻️
_Check the comments under the review._

- Kindly a video not than 5mins presenting the features of your project 
- The live link provided seem broken kindly fix that too

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hi there @Gleennkar👋🏽 , 

Great work on making the changes requested by a previous reviewer 👏 
To highlight 👍🏽 
- There is no longer an overflow on your page.
- Commented code has been removed.
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Required Changes ♻️
- [ ] Like the previous reviewer stated, kindly record a video presenting your project, as it is a part of the requirements.
- [ ] The live link in your README currently gives a 404 error, please fix it.
- [ ] Kindly ensure that the products section of your page has the same design as the featured speaker section as the previous reviewer mentioned. Also, consider making your footer in the mobile version look the some as the original design as the last reviewer also mentioned.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @Gleennkar, 

## Status: APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 
- Great job with your design and functionalities 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**